src/policy/fees.h

```diff
@@ -14,12 +14,17 @@
 #include <util/fs.h>
 
 #include <array>
+#include <chrono>
 #include <map>
 #include <memory>
 #include <set>
 #include <string>
 #include <vector>
 
+
+// How often to flush fee estimates to fee_estimates.dat.
+static constexpr std::chrono::milliseconds FEE_FLUSH_INTERVAL{std::chrono::hours{1}};
```

Comment:
I don't actually know for sure but couldn't this just be ...?

```cpp
static constexpr std::chrono::hours FEE_FLUSH_INTERVAL{1};
```

Reply:
Still a nub on cpp,
 `constexpr auto FEE_FLUSH_INTERVAL{1h};`  does not compile
I added instead `static constexpr std::chrono::hours FEE_FLUSH_INTERVAL{1};`